scripts/merge_hhs_nppes.py

```
@@ -1,7 +1,11 @@
 """Merge HHS Medicaid dental spending against NPPES practice-location data,
 one month at a time, without ever loading the full NPPES corpus into memory.
 
-Replaces the SQL join in migrations/005_populate_provider_procedure_monthly_geo.sql.
+The merge runs in Python rather than SQL because NPPES archives use
+deflate64 (needs 7-Zip, not stdlib zipfile) and a per-month streaming
+flow is dramatically cheaper than loading 84 × ~9M-row vintages into
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)